File: cpp_e9zMZ/cpp/10-tonghop/123.cpp
#include <iostream>
using namespace std;

int main() {
    int num = 92;
    int sum = 0;

    while (num > 0) {
        int lastNumber = num % 10;
        sum += lastNumber;
        num = num / 10;
        if (num == 0 && sum > 10) {
            num = sum;
            sum = 0;
        }
    }

    cout << sum;
    return 0;
}

File: python_ToSXs/python/10-tonghop/123.py
num = 99992

zsum = 0

while num > 0:
    lastNumber = num % 10

    zsum += lastNumber
    num = int(num / 10)

    if num == 0 and zsum > 10:
        num = zsum
        zsum = 0

print(zsum)
